src/utils/__tests__/csvValidation.test.ts

```
@@ -147,7 +149,8 @@ describe('csvValidation', () => {
     it('truncates at word boundary when possible', () => {
       const longTag =
         'This is a very long tag name that should be truncated at a good word boundary if possible and exceed the maximum field length significantly';
-      const longTag = 'This is a very long tag name that should be truncated at a good word boundary if possible and exceed the maximum field length significantly';
+      const longTag =
```

Biome (1.9.4)

[error] Shouldn't redeclare 'longTag'. Consider to delete it or rename it.

'longTag' is defined here:


(lint/suspicious/noRedeclare)